src/backend/base/langflow/api/v1/mcp_projects.py

```
@@ -311,50 +306,43 @@ async def send_progress_updates():
 
                 collected_results = []
                 try:
-                    progress_task = asyncio.create_task(send_progress_updates())
+                    progress_task = None
+                    if mcp_config.enable_progress_notifications and self.server.request_context.meta.progressToken:
```

Pylint (3.3.7)

[convention] Line too long (115/100)

(C0301)